lib/repl.js

```diff
@@ -1126,7 +1126,7 @@ function isRecoverableError(e, self) {
       self._inTemplateLiteral = true;
       return true;
     }
-    return /^(Unexpected end of input|Unexpected token)/.test(message);
+    return /^(Unexpected end of input|Unexpected token|missing \) after argument list)/.test(message);
```

Comment:
I know, but you can break it up into multiple regular expressions or compile one dynamically with `new RegExp(...)`.  I'd take the multi-regex approach.